engine-rest/engine-rest-openapi/src/main/templates/paths/history/activity-instance/count/get.ftl

```diff
@@ -0,0 +1,37 @@
+<#assign requestMethod="GET"/>
+<#include "/lib/commons/history-activity-instance.ftl" >
```

Comment:
Is it possible to move this to the parameters?

Reply:
👍